Cargo.toml

```diff
@@ -27,17 +32,17 @@ blst = { version = "0.3.11", default-features = false }
 hex = { version = "0.4.2", default-features = false, features = ["alloc"] }
 libc = { version = "0.2", default-features = false }
 serde = { version = "1.0", optional = true, default-features = false, features = [
-    "alloc",
-    "derive",
+	"alloc",
+	"derive",
```

Comment:
not a big deal, but do we need to change the whitespace here?

Reply:
Restored original whitespaces.